vertical-tabbrowser.xml

```diff
@@ -220,6 +191,21 @@
           draggedTab.style.transform = 'translateY(' + translateVal + 'px)';
           // console.log('draggedTab transform: ' + draggedTab.style.transform + ' should be ' + 'translateY(' + translateVal + 'px)');
 
+          let tabbrowser = document.getElementById('tabbrowser-tabs');
+          let scrollbox = document.getAnonymousElementByAttribute(tabbrowser, 'anonid', 'arrowscrollbox');
+          let scrollbuttonDown = document.getAnonymousElementByAttribute(scrollbox, 'anonid', 'scrollbutton-down');
+          let scrollbuttonUp = document.getAnonymousElementByAttribute(scrollbox, 'anonid', 'scrollbutton-up');
+          let mouseY = event.clientY;
+
+          if (mouseY > (document.getElementById('tabbrowser-tabs').boxObject.height + 10)){
+            scrollbuttonDown.onmousedown(event);
+          } else if (mouseY < 65) {
```

Comment:
![scrollpinned](https://cloud.githubusercontent.com/assets/10803178/18685548/673d0d28-7f2d-11e6-9955-3095a12603ea.gif)


no... it's not that...